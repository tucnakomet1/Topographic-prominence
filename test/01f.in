8054 6219 6085 6593 6093 8736 8269 7169 6392 7731 7400 8363 6343 6279 8041