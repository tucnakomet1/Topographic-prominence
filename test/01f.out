1969
500
8736
331
1971
1762